File: modules/core/src/main/java/kr/entree/enderwand/command/Command.java
package kr.entree.enderwand.command;

import kr.entree.enderwand.command.completer.TabCompleter;
import kr.entree.enderwand.command.executor.CommandExecutor;
import kr.entree.enderwand.command.sender.Sender;

/**
 * Created by JunHyung Lim on 2020-02-04
 */
public interface Command<S extends Sender> extends CommandExecutor<S>, TabCompleter<S> {
}

File: modules/core/kotlin/src/main/kotlin/kr/entree/enderwand/command/Command.kt
package kr.entree.enderwand.command

import kr.entree.enderwand.command.executor.Executor
import kr.entree.enderwand.command.sender.Sender
import kr.entree.enderwand.command.tabcompleter.TabCompleter

/**
 * Created by JunHyung Lim on 2019-12-18
 */
interface Command<S : Sender> : Executor<S>, TabCompleter<S>